01-intro/01-introduction/04-quiz/README.md

```diff
@@ -0,0 +1,121 @@
+# Quiz 1
+
+- Tipo: `quiz`
+- Formato: `self-paced`
+- Duración: `20min`
+
+***
+
+## Preguntas
+
+### 1) Cada navegador web viene con un intérprete JavaScript que entiende cómo leer programas JavaScript
+
+#### Opciones
+
+1. Verdadero
+2. Falso
+
+<solution style="display:none;">1</solution>
+
+### 2) JavaScript es una versión moderna del lenguaje de programación Java
+
+#### Opciones
+
+1. Verdadero
+2. Falso
+
+<solution style="display:none;">2</solution>
+
+### 3) ¿Cuál de los siguientes NO es un navegador web?
+
+#### Opciones
+
+1. Google Chrome
+2. Internet Explorer
+3. Safari
+4. JavaScript
+
+<solution style="display:none;">4</solution>
+
+### 4) HTML es ...*
```

Comment:
¿Porqué el `*` al final?

Reply:
Para evitar el error del mdlint   [MD026 - Trailing punctuation in header] (https://github.com/DavidAnson/markdownlint/blob/HEAD/doc/Rules.md#md026)

Se puede modificar para que quede de la mejor manera.